tests/keras/backend/backend_test.py

```diff
@@ -1425,28 +1425,37 @@ def test_bias_add(self):
         with pytest.raises(ValueError):
             K.bias_add(x, b, data_format='channels_middle')
 
-    def test_batchnorm(self):
```

Comment:
The skipping looks reasonable, however, it is hard for me to see improvements in these refactoring...